charts-external/openpension/healthcheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+kubectl rollout status deployment/openpension
```

Comment:
should change it to check both deployments (client / server) e.g. 

kubectl rollout status deployment/openpension-client && kubectl rollout status deployment/openpension-server